app/public/static/js/components/hint-button.js

```diff
@@ -24,25 +24,25 @@ AFRAME.registerComponent("hint-button", {
     this.hintIcon.object3D.position.set(x, y, z);
 
     // Create hint label
-    this.hintLabel = document.createElement("a-text");
-    this.hintLabel.setAttribute("id", "hint-label");
-    this.hintLabel.setAttribute("value", "New Hints!");
-    this.hintLabel.setAttribute("color", "black");
-    this.hintLabel.setAttribute("scale", { x: 0.07, y: 0.07, z: 0.07 });
-    this.hintLabel.setAttribute("align", "center");
-    this.hintLabel.setAttribute(
-      "font",
-      "https://raw.githubusercontent.com/jaydhulia/aframe-fonts/master/fonts/poppins/Poppins-Bold.json"
-    );
-    this.hintLabel.setAttribute("shader", "msdf");
-    this.hintLabel.object3D.position.set(x, y - (data.height * 2) / 2.6, z);
-    this.el.appendChild(this.hintLabel);
+    // this.hintLabel = document.createElement("a-text");
+    // this.hintLabel.setAttribute("id", "hint-label");
+    // this.hintLabel.setAttribute("value", "New Hints!");
+    // this.hintLabel.setAttribute("color", "black");
+    // this.hintLabel.setAttribute("scale", { x: 0.07, y: 0.07, z: 0.07 });
+    // this.hintLabel.setAttribute("align", "center");
+    // this.hintLabel.setAttribute(
+    //   "font",
+    //   "https://raw.githubusercontent.com/jaydhulia/aframe-fonts/master/fonts/poppins/Poppins-Bold.json"
+    // );
+    // this.hintLabel.setAttribute("shader", "msdf");
+    // this.hintLabel.object3D.position.set(x, y - (data.height * 2) / 2.6, z);
+    // this.el.appendChild(this.hintLabel);
```

Comment:
Let's remove commented out code, unless this is needed for future (in which case let's leave a comment indicating that!)

Reply:
oh yes. I forgot to remove